pkg/cidata/cidata.TEMPLATE.d/boot/05-rosetta-volume.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+
+if [ ! -d /tmp/rosetta-share ]; then
```

Comment:
What's /tmp/rosetta-share ?

Reply:
It is the the "Rosetta Directory Share" (see https://developer.apple.com/documentation/virtualization/running_intel_binaries_in_linux_vms_with_rosetta).

The sample code mounts it under `/tmp`, but I would prefer to mount it as `/mnt/rosetta` instead.

Reply:
Oops, I just forgot to update the path in the conditional to match the path where I mounted it.